tests/e2e/test_online_features.py

```diff
@@ -128,3 +137,111 @@ def test_offline_ingestion(feast_client: Client, staging_path: str):
             columns={"unique_drivers": "drivers:unique_drivers"}
         ),
     )
+
+
+def test_streaming_ingestion(feast_client: Client, staging_path: str, pytestconfig):
+    entity = Entity(name="s2id", description="S2id", value_type=ValueType.INT64,)
+
+    feature_table = FeatureTable(
+        name="drivers_stream",
+        entities=["s2id"],
+        features=[Feature("unique_drivers", ValueType.INT64)],
+        batch_source=FileSource(
+            "event_timestamp",
+            "event_timestamp",
+            ParquetFormat(),
+            os.path.join(staging_path, "batch-storage"),
+        ),
+        stream_source=KafkaSource(
+            "event_timestamp",
+            "event_timestamp",
+            pytestconfig.getoption("kafka_brokers"),
+            AvroFormat(avro_schema()),
+            topic="avro",
+        ),
+    )
+
+    feast_client.apply_entity(entity)
+    feast_client.apply_feature_table(feature_table)
+
+    job = feast_client.start_stream_to_online_ingestion(feature_table)
+
+    wait_retry_backoff(
+        lambda: (None, job.get_status() == SparkJobStatus.IN_PROGRESS), 60
+    )
+
+    time.sleep(5)
+
+    original = generate_data()[["s2id", "unique_drivers", "event_timestamp"]]
+    for record in original.to_dict("records"):
+        record["event_timestamp"] = int(record["event_timestamp"].timestamp() * 1e6)
```

Comment:
Based on the error message here: http://prow.feast.ai/view/gcs/feast-templocation-kf-feast/pr-logs/pull/feast-dev_feast/1072/test-end-to-end/1318191468216586242

It seems like a `datetime` object input is expected, which will then be converted to long.